test/ppx_irmin/deriver/passing/extension.ml

```diff
@@ -0,0 +1,29 @@
+type 'a typ = 'a Irmin.Type.t
+
+module Simple = struct
+  let (_ : (int * string) list typ) = [%typ: (int * string) list]
```

Comment:
I'm wondering why we need to inline the derived type representation. It's not very clean to use the derived `test_t` (I think this is what we have now for a type `test`)?